src/plugins/Wallet/database/types.ts

```diff
@@ -91,8 +91,8 @@ export interface RedPacketRecordInDatabase
     shares: string | bigint
 }
 export interface WalletRecordInDatabase extends Omit<WalletRecord, 'eth_balance' | 'erc20_token_balance'> {
-    eth_balance?: string | bigint
-    erc20_token_balance: Map<string, string | bigint | undefined>
+    eth_balance: string | bigint
+    erc20_token_balance: Map<string, string | bigint>
```

Comment:
Even you don't write new `undefined` value in, you need to keep that type or you will forget to handle old data which might be undefined.

Reply:
All of the wallet records will be handled before out DB. These fields will be added to old records.